stockAndPrices/services/app/games.py

```diff
@@ -0,0 +1,125 @@
+from flask import Flask, Blueprint, request
+import requests, json
+
+games_blueprint = Blueprint("games", __name__, url_prefix="/games")
+api_url = "https://www.cheapshark.com/api/1.0"
+
+
+
+#### Games ####
+
+# /title?name=somename
+@games_blueprint.route('/title')
+def gamePriceByTitle():
+    game_title = validation(request.args.get('name', ''))
+    url = api_url + f"/games?title={game_title}"
+    payload, files, headers = {}, {}, {}
+    response = requests.request("GET", url, headers=headers, data=payload, files=files)
+    if response.status_code != 200:
+        return f"Get request from {url} failed.", 500
+    resp = json.loads(response.text)
+    all_ids = ''
+    all_prices = {}
+    count = 0
+    # retrieve all gameids from searching that game
+    for rs in resp:
+        if not all_ids:
+            all_ids += rs['gameID']
+        else:
+            all_ids += ',' + rs['gameID']
+        count += 1
+        if count == 25:
+            findGamePriceById(all_ids, all_prices)
+            count, all_ids = 0, ''
+
+    if not all_ids and not all_prices:
```

Comment:
Would we able to determine there are no results before executing the loop on line 25-33?

Reply:
Yes, updated. If resp is empty then there is no result